app/helpers/macros/__init__.py

```diff
@@ -0,0 +1,2 @@
+# This contains the macros to pass to a rendering instance
+from app.helpers.macros.gravatar import *
```

Comment:
Not sure if this can be golfed. `__all__ = 'gravatar'` didn't work

Reply:
`__all__ = ['gravatar']` - `__all__` is expected to be a list.

Reply:
yeah, sorry that's what I meant